jest.config.js

```diff
@@ -1,11 +1,19 @@
+const transformIgnorePatterns = ['node_modules/(?!(@openshift|lodash-es)/)'];
```

Comment:
FYI, all OpenShift plugin SDK packages, except the webpack one, currently ship with ESM build output.

We plan to update these packages to provide a CommonJS build output, in addition to ESM.